core/src/main/scala/org/bitcoins/core/wallet/fee/FeeUnit.scala

```diff
@@ -35,6 +35,13 @@ sealed abstract class FeeUnit {
   def toLong: Long = currencyUnit.satoshis.toLong
 
   override def toString: String = s"$toLong ${factory.unitString}"
+
+  def toSatsPerVByte: SatoshisPerVirtualByte = this match {
+    case s: SatoshisPerVirtualByte => s
+    case s: SatoshisPerByte        => s.toSatsPerVByte
+    case s: SatoshisPerKiloByte    => s.toSatsPerVByte
+    case s: SatoshisPerKW          => s.toSatsPerVByte
+  }
```

Comment:
won't this just infinitely loop

Reply:
for example `SatoshisPerKiloByte` doesn't have this implemented

Reply:
i am not a smart man :sweat_smile: